constants/event_flags.asm

```diff
@@ -577,7 +577,7 @@
 	const EVENT_BEAT_CAMPER_LLOYD
 	const EVENT_BEAT_CAMPER_DEAN
 	const EVENT_BEAT_CAMPER_SID
-	const EVENT_BEAT_CAMPER_HERVEY
+	const EVENT_BEAT_CAMPER_HARVEY
```

Comment:
Since this event is not used anywhere, it should be `const_skip`. Same for `EVENT_BEAT_SCHOOLBOY_KIPP`.